paystell-frontend/src/services/wallet-transaction.service.ts

```
@@ -49,16 +50,81 @@ export const fetchTransactionHistory = async (
+      if (filters.minAmount || filters.maxAmount) {
+        // Note: This is a simplified implementation as transaction amounts would need to be 
+        // extracted from operations, which would require additional API calls per transaction
+        // For a complete implementation, you'd need to fetch operations for each transaction
+      }
+
+      // Search filtering (simplified implementation)
+      if (filters.searchQuery) {
+        const query = filters.searchQuery.toLowerCase()
+        records = records.filter(tx => 
+          tx.id.toLowerCase().includes(query) || 
+          (tx.memo && tx.memo.toLowerCase().includes(query))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)